.travis.yml

```diff
@@ -83,9 +86,10 @@ script:
     - docker ps --all
     - docker images
     - docker rmi localhost:5000/my-alpine
+    - docker logout localhost:5000
     # Setting up  Docker Registry is complete, let's do Buildah testing! 
     - make install.tools install.libseccomp.sudo all runc validate SECURITYTAGS="apparmor seccomp"
-    - go test -c -tags "apparmor seccomp `./btrfs_tag.sh` `./libdm_tag.sh` `./ostree_tag.sh` `./selinux_tag.sh`" ./cmd/buildah
+    - go test -coverprofile=coverage.txt -covermode=atomic -c -tags "apparmor seccomp `./btrfs_tag.sh` `./libdm_tag.sh` `./ostree_tag.sh` `./selinux_tag.sh`" ./cmd/buildah
```

Comment:
I guess this line is only for building the test binary, the actual execution follow below.